tools/openms/.shed.yml

```diff
@@ -0,0 +1,12 @@
+name: openms
+owner: galaxyp

```

Comment:
More like this: https://github.com/galaxyproject/tools-iuc/blob/master/tools/gemini/.shed.yml
So that every tool ends up in its own toolshed package.